Sources/ComposableArchitecture/Reducer.swift

```diff
@@ -266,9 +266,9 @@ public struct Reducer<State, Action, Environment> {
   /// Transforms a reducer that works on non-optional state into one that works on optional state by
   /// only running the non-optional reducer when state is non-nil.
   ///
-  /// Often used in tandem with `pullback` to transform a reducer on a non-optional local domain
-  /// into a reducer that can be combined with a reducer on a global domain that contains some
-  /// optional local domain:
+  /// Often used in tandem with `pullback` to transform a reducer on a non-optional child domain
+  /// into a reducer that can be combined with a reducer on a parent domain that contains some
+  /// optional child domain:
```

Comment:
Any thoughts on updating our terminology from global-local to parent-child? It read better to me here. Global alone sounds like app-level domain, so you need to qualify as "more global" or "more local"...